src/Validation/Report.cs

```diff
@@ -4,9 +4,10 @@
 namespace Validation
```

Comment:
This class has been totally broken. So while we're retrofitting it, let's fix the class. We need to add [these lines](https://github.com/microsoft/vs-validation/blob/f29f307c61f28bd401b7ea337e9067fbe0c60773/src/Microsoft.VisualStudio.Validation/Report.cs#L4-L6) to this file. Otherwise when we compile a release build of this library, this class is completely useless.

Reply:
Strange. I *thought* I'd pulled those over. I'll update it.